deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -7,6 +7,8 @@ services:
       - ./clickhouse-config.xml:/etc/clickhouse-server/config.xml
       - ./data/clickhouse/:/var/lib/clickhouse/
     restart: on-failure
+    ports:
+      - "9000:9000"
```

Comment:
why is clickhouse port exposed? I don't think it is needed

Reply:
I thought you suggested it in last meeting. I thought it was needed on staging. will remove it.

Reply:
Changes
- Removed the port exposure for click house 
- changed link input for alert manager from localhost to query-service
- moved alert manager after query-service to avoid runtime dependency issue